tools/tests/unit/test_spice_examples.py

```diff
@@ -0,0 +1,82 @@
+from pathlib import Path
+
+import numpy as np
+import pytest
+import spiceypy as spice
+
+from tools.spice.spice_examples import (
+    _get_particle_velocity,
+    build_annotated_events,
+    get_attitude_timerange,
+)
+from tools.spice.spice_utils import (
+    list_files_with_extensions,
+)
+
+
+@pytest.fixture()
+def kernel_directory():
+    """Kernel directory."""
+    kernel_directory = Path(__file__).parents[1] / "test_data" / "spice"
+    return kernel_directory
+
+
+@pytest.fixture()
+def kernels(kernel_directory):
+    """Loads all kernels."""
+    # TODO: ALl kernels able to be downloaded from NAIF are not available
+    #  in the test_data/spice directory.
+    kernels = list_files_with_extensions(
+        kernel_directory, [".tsc", ".tls", ".tf", ".bsp", ".ck"]
+    )
+    return kernels
+
+
+@pytest.fixture()
+def direct_events():
+    """Loads direct events test data.
+
+    Possible example of future direct_events structure:
+    direct_events.keys()
+    dict_keys(['eventID', 'tdb', 'xstart', 'ystart', 'xstop', 'ystop',
+    'eventtype', 'ph', 'spinphase', 'tof', 'ishead90', 'yf', 'd',
+    'energy', 'rvecultra', 'vultra', 'rmagultra', 'rnormultra'])
+    """
+    direct_events = {
+        "vultra": np.array(
+            [-5.7718470e08, -8.8437594e08, -9.5545958e08], dtype=np.float32
+        ),
+        "tdb": 802100000.0,
+    }
+
+    return direct_events
+
+
+@pytest.mark.xfail(reason="Download NAIF kernels")
```

Comment:
Nice, thanks for adding these messages and xfails to indicate what we need to do to fix them up later!